testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/util/IcebergTestUtils.java

```diff
@@ -34,4 +38,9 @@ public static String getTableLocation(String tableName)
         }
         throw new IllegalStateException("Location not found in SHOW CREATE TABLE result");
     }
+
+    public static String getTableLocation(String tableName)
```

Comment:
per code style: this would go above the more detailed version

Reply:
Reverted my change.